ui/apps/platform/src/constants/networkFlow.ts

```diff
@@ -0,0 +1,11 @@
+import { L4Protocol } from 'types/networkFlow.proto';
+
+export const l4ProtocolLabels: Record<L4Protocol, string> = {
```

Comment:
There is a partial implementation of this under `src/messages/network.js`. I opted to leave it for now since most (all?) of that file will disappear along with `src/constants/network.*.js` and the related network sagas.